### PR TITLE
Affinity

### PR DESCRIPTION
**Pull request by srueg**

### CODE REVIEW

File: stardog/values.yaml [hunk 1]
```diff
@@ -83,3 +83,13 @@ zookeeper:
     # serverPasswords: secretPassword,
   metrics:
     enabled: true
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
```
Comment (anothertobi) [inline stardog/values.yaml]: We could add `{{ .Release.Name }}` for the `release`label in order to allow multiple zookeepers in the same namespace.

Comment (srueg) [reply stardog/values.yaml]: Good catch but it's unfortunately not possible to use variables in the `values.yaml` file.

Comment (srueg) [reply stardog/values.yaml]: But I found another issue.

Comment (srueg) [reply stardog/values.yaml]: Ok, it's solved.

**Pull request merged**